### DISCUSSION TITLE
[Bug] page content not showing on docmost upgrade

### DISCUSSION THREAD

**Discussion by Gandalf-Le-Dev** [docmost/docmost#252] [Help / Questions] [answered]
I upgraded to the latest version to get the excalidraw feature but I lost all my files content. What should I do ? Are they deleted definitely or is it not fetched correctly ? How can i check that ?

How did I upgrade docmost ? Like the doc says 
```
docker pull docmost/docmost
docker compose up --force-recreate --build docmost -d
```

**Comment by Philipinho** [comment, marked as answer]
Please upgrade to [v0.3.1](https://github.com/docmost/docmost/releases/tag/v0.3.1). That should fix it.

**Reply by Gandalf-Le-Dev** [reply]
Indeed
